replicate-to-sql-with-generic-events/scripts/delete-data.js

```diff
@@ -0,0 +1,47 @@
+const { Kuzzle, WebSocket } = require('kuzzle-sdk');
+
+const kuzzle = new Kuzzle(new WebSocket('localhost'));
+
+const indexName = 'nyc-open-data';
+const collectionName = 'yellow-taxi';
+
+async function searchData() {
+  try {
+    const result = await kuzzle.document.search(
+      indexName,
+      collectionName,
+      {},
+      { size: 500 }
+    );
+    return result.hits;
+  } catch (error) {
```

Comment:
Nitpicking: wrong coding standard
Also, IMHO an error from Kuzzle should not be silenced here because it can confuse the user on why his documents are not deleted